vllm/core/block/naive_block.py

```diff
@@ -36,7 +36,8 @@ def __init__(
         if block_ids is None:
             block_ids = range(num_blocks)
 
-        self._free_block_indices: Deque[BlockId] = deque(block_ids)
+        self._free_block_indices: List[BlockId] = list(block_ids)
```

Comment:
Please add some comments to illustrate why need such changes since it's in common path.